src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -65,6 +67,13 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.STR_DPKG_WAS_INTERRUPTED = "E: dpkg was interrupted, you must manually run 'sudo dpkg --configure -a' to correct the problem."
         self.ESM_MARKER = "The following packages could receive security updates with UA Infra: ESM service enabled:"
 
+        # Ubuntu Pro Client pre-requisite checks.
+
+        # This flag will be used to determine if Ubuntu Pro Client can be used for querying reboot status or get packages list.
```

Comment:
nit: this comment only applies to 73 and not the whole "paragraph", so it's better to append to the end of the line that it applies to (line 73)

Reply:
Sure. Will do.